ADO.NET/OAuthSampleApp/OAuthSampleApp.cs

```diff
@@ -20,21 +22,18 @@ public static void ConnectToDatabase(string connectionString)
         {
             using (VerticaConnection conn = new VerticaConnection(connectionString))
             {
+                Console.WriteLine("Attempting to connect with connection string: " + ResultSetPrinter.printableConnectionString(connectionString));
                 conn.Open();
 
                 using (VerticaCommand command = conn.CreateCommand())
                 {
                     Console.WriteLine("Connection Successful");
-                    command.CommandText = "SELECT user_id, user_name FROM USERS ORDER BY user_id";
-                    using (VerticaDataReader dataReader = command.ExecuteReader())
+                    command.CommandText = ConfigurationManager.AppSettings["Query"];
+                    using (VerticaDataReader reader = command.ExecuteReader())
                     {
-                        Console.WriteLine("Executing Query");
-                        int rowIdx = 1;
-                        while (dataReader.Read())
-                        {
-                            Console.WriteLine(rowIdx + ". " + dataReader.GetString(0).Trim() + " " + dataReader.GetString(1).Trim());
-                            rowIdx++;
-                        }
+                        Console.WriteLine("Executing Query: " + command.CommandText);
+                        ResultSetPrinter printer = new ResultSetPrinter(reader);

```

Comment:
I just copied the ResultSetPrinter from the sample app exactly. It works fine here.

Reply:
Could the samples use a common ResultSetPrinter in something like a Utils folder?  I don't like having two copies of the same source.